Tweak xvnc RandR values to take into account the trust bar

Distributed under the terms of the GNU General Public License v2

diff --git a/unix/xserver/hw/vnc/xvnc.cc b/unix/xserver/hw/vnc/xvnc.cc
index 73bb6cf..1697b39 100644
--- a/unix/xserver/hw/vnc/xvnc.cc
+++ b/unix/xserver/hw/vnc/xvnc.cc
@@ -1277,8 +1277,8 @@ static void vncRandRModeDestroy(ScreenPtr pScreen, RRModePtr mode)
     /* We haven't allocated anything so nothing to destroy */
 }
 
-static const int vncRandRWidths[] =  { 1920, 1920, 1600, 1680, 1400, 1360, 1280, 1280, 1280, 1280, 1024, 800, 640 };
-static const int vncRandRHeights[] = { 1200, 1080, 1200, 1050, 1050,  768, 1024,  960,  800,  720,  768, 600, 480 };
+static const int vncRandRWidths[] =  { 2522, 2522, 1882, 1882, 1562, 1562, 1642, 1362, 1402, 1328, 1242, 1242, 1242, 1242, 1114, 986, 986, 762, 602 };
+static const int vncRandRHeights[] = { 1594, 1434, 1194, 1074, 1194,  894, 1044, 1044,  894,  762, 1018,  954,  794,  714,  858, 762, 594, 594, 474 };
 
 static int vncRandRIndex = 0;
 
